sys/include/net/bluetil/addr.h

```diff
@@ -55,6 +55,17 @@ void bluetil_addr_sprint(char *out, const uint8_t *addr);
  */
 void bluetil_addr_print(const uint8_t *addr);
 
+/**
+ * @brief   Parse a BLE address from the given string
+ *
+ * @param[in] add_str   address string, must be at least of length
```

Comment:
Travis reports: `addr` parameter missing. Also (nit-picky) why not `addr_str`?

Reply:
fixed. The `add_str` was simply a typo, in the c file I already used `addr_str` :-)